opensaml/3.3.1.wso2v3/pom.xml

```diff
@@ -0,0 +1,1151 @@
+                            @opensaml-storage-impl-${opensaml.version}.jar!/conf/**,
+
+                            @opensaml-xacml-api-${opensaml.version}.jar!/**.xml,
+                            @opensaml-xacml-api-${opensaml.version}.jar!/templates/**.vm,
+                            @opensaml-xacml-api-${opensaml.version}.jar!/schema/**.xsd,
+                            @opensaml-xacml-api-${opensaml.version}.jar!/conf/**,
+
+                            @opensaml-xacml-impl-${opensaml.version}.jar!/**.xml,
+                            @opensaml-xacml-impl-${opensaml.version}.jar!/templates/**.vm,
+                            @opensaml-xacml-impl-${opensaml.version}.jar!/schema/**.xsd,
+                            @opensaml-xacml-impl-${opensaml.version}.jar!/conf/**,
+
+                            @opensaml-xacml-saml-api-${opensaml.version}.jar!/**.xml,
+                            @opensaml-xacml-saml-api-${opensaml.version}.jar!/templates/**.vm,
+                            @opensaml-xacml-saml-api-${opensaml.version}.jar!/schema/**.xsd,
+                            @opensaml-xacml-saml-api-${opensaml.version}.jar!/conf/**,
+
+                            @opensaml-xacml-saml-impl-${opensaml.version}.jar!/**.xml,
+                            @opensaml-xacml-saml-impl-${opensaml.version}.jar!/templates/**.vm,
+                            @opensaml-xacml-saml-impl-${opensaml.version}.jar!/schema/**.xsd,
+                            @opensaml-xacml-saml-impl-${opensaml.version}.jar!/conf/**,
+
+                            @opensaml-xmlsec-api-${opensaml.version}.jar!/**.xml,
+                            @opensaml-xmlsec-api-${opensaml.version}.jar!/templates/**.vm,
+                            @opensaml-xmlsec-api-${opensaml.version}.jar!/schema/**.xsd,
+                            @opensaml-xmlsec-api-${opensaml.version}.jar!/conf/**,
+                            @opensaml-xmlsec-api-${opensaml.version}.jar!/META-INF/services/**,
+
+                            @opensaml-xmlsec-impl-${opensaml.version}.jar!/**.xml,
+                            @opensaml-xmlsec-impl-${opensaml.version}.jar!/templates/**.vm,
+                            @opensaml-xmlsec-impl-${opensaml.version}.jar!/schema/**.xsd,
+                            @opensaml-xmlsec-impl-${opensaml.version}.jar!/conf/**,
+                            @opensaml-xmlsec-impl-${opensaml.version}.jar!/META-INF/services/**,
+
+                            @java-support-${shibboleth.version}.jar!/**.xml,
+                            @java-support-${shibboleth.version}.jar!/templates/**.vm,
+                            @java-support-${shibboleth.version}.jar!/schema/**.xsd,
+                            @java-support-${shibboleth.version}.jar!/conf/**,
+                        </Include-Resource>
+                    </instructions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    <properties>
```

Comment:
Better to leave a line between </build> and <properties>

Reply:
Fixed with https://github.com/wso2/orbit/pull/492